frontend/tailwind.config.js

```diff
@@ -58,6 +58,10 @@ module.exports = {
 			'table-padding': '.5rem',
 			'primary-btn-padding-x': '2rem',
 			'primary-btn-padding-y': '1rem',
+			'hamburger-w': '2rem',
+			'hamburger-h': '1.4rem',
+			'hamburger-line-w': '.2rem',
+			'menu-spacing': '1rem'
```

Comment:
I sincerely don't like this approach.  
The tailwind key names should be short and not repetitive.
1: '1rem'
2: '2rem'
etc

did you know that for each spacing declared here, tailwind will generate a lot of classes?
This will make the CSS development file very big and if it gets too big it will have performance consequences even in local.

Reply:
I don't think performance will be an issue due to purgecss (which really ought to run on local too. The advantage I see here is that you can adjust each property without having to go back and find where it's used in the HTML, particularly useful when setting up a new project base styles.

Reply:
You can see Adam W has done something similar in this example: https://github.com/adamwathan/theming-tailwind-demo/blob/master/tailwind.config.js#L29

I do get it may be overkill for things like the hamburger width/height which really are only going to be used once on that specific element. Maybe we can think of some intermediate solution which provides the best of both ideas?

Reply:
are you proposing that for each spacing needed for every component we need to create the spacing here?

Reply:
yes, i guess i am. i was assuming that there would be a limited number of spacing options, we wouldn't create spacing for each component, but more general - eg "card-padding" would be the same for all cards. But if this isn't going to be too much then we have to rethink it.